cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -970,16 +956,16 @@ private String queryByFt(String index, boolean hasIndex, IndexTextAndJsonValues
     // Clean the term. Remove stand-alone ' *', not valid word.
```

Comment:
Thank you for removing  /respectAccents and /respectCase from our regular unit tests.
However, we need to ensure that the logger warnings "Ignoring /respectAccents" "Ignoring /respectCase" work and we do not throw an exception in these cases. Can you add unit tests that check this and cover those lines?